src/state/scigateway.types.tsx

```diff
@@ -188,6 +188,7 @@ export interface ScheduledMaintenanceStatePayLoad {
 export interface ScheduledMaintenanceState {
   show: boolean;
   message: string;
+  severity: 'success' | 'warning' | 'error' | 'information';
```

Comment:
For backwards compatibility - we should probably allow this to be undefined as well, and handle that logic to assume it means warning as otherwise this could cause problems when upgrading. For example, deleting the `severity` from the SG preprod backend JSON file means a toast won't display - despite `show` being true. All the facility preprod/prod machines don't have `severity` in the JSON.

On the other hand, if it makes the code more complicated unecessarily, we can change https://github.com/ral-facilities/scigateway-auth/blob/main/maintenance/scheduled_maintenance.json (i.e. the default file) and I can manually edit all the existing files to have warning severity.